samples/dashboards/sync/custom-component-sync/demo.js

```diff
@@ -0,0 +1,286 @@
+const { Component, ComponentRegistry } = Dashboards;
+
+class AveragesMirror extends Component {
+    constructor(cell, options) {
+        super(cell, options);
+        this.type = 'AveragesMirror';
+        this.createDOMStructure();
+        this.xColumn = [];
+        this.yColumn = [];
+        this.sync = new Component.Sync(
```

Comment:
This part is also crucial for sync to work- we should desire that if someone wants to create custom component with sync what should be done.